truss-chains/truss_chains/deployment/code_gen.py

```diff
@@ -792,14 +815,23 @@ def gen_truss_chainlet(
         f"Code generation for {chainlet_descriptor.chainlet_cls.entity_type} `{chainlet_descriptor.name}` "
         f"in `{chainlet_dir}`."
     )
+    if issubclass(chainlet_descriptor.chainlet_cls, framework.EngineBuilderChainlet):
+        engine_builder_config = chainlet_descriptor.chainlet_cls.engine_builder_config
+    else:
+        engine_builder_config = None
+
     _write_truss_config_yaml(
         chainlet_dir=chainlet_dir,
         chains_config=chainlet_descriptor.chainlet_cls.remote_config,
         model_name=model_name or chain_name,
         chainlet_to_service=dep_services,
         use_local_chains_src=use_local_chains_src,
         is_websocket_endpoint=chainlet_descriptor.endpoint.is_websocket,
+        engine_builder_config=engine_builder_config,
     )
+    if engine_builder_config:
```

Comment:
Just for my understanding, does this mean that chainlets that use engine builder cannot have python [code](https://docs.baseten.co/performance/engine-builder-customization) on top? I think that's fine, but might be worth a comment